src/CMakeLists.txt

```diff
@@ -38,35 +38,25 @@ macro (upm_target_link_libraries target_name)
 
     # Call the swig link library method for each library
     if (BUILDSWIGNODE)
-      set_target_properties(${SWIG_MODULE_jsupm_${libname}_REAL_NAME} PROPERTIES SKIP_BUILD_RPATH TRUE)
       swig_link_libraries (jsupm_${libname} ${_library})
```

Comment:
This is only for node modules thus Eclipse wouldn't be impacted. However, using SKIP_BUILD_RPATH TRUE will likely remove the path for both build & install. Ideally we want the path for the build (in case tests need to run) and only stripped when the module is installed.